freqtrade/rpc/rpc.py

```diff
@@ -136,7 +136,10 @@ def _rpc_show_config(config, botstate: Union[State, str],
             'ask_strategy': config.get('ask_strategy', {}),
             'bid_strategy': config.get('bid_strategy', {}),
             'state': str(botstate),
-            'runmode': config['runmode'].value
+            'runmode': config['runmode'].value,
+            'position_adjustment_enable': config.get('position_adjustment_enable', False),
+            'max_buy_position_adjustment': (config['max_buy_position_adjustment']
```

Comment:
I do appreciate adding this here - please also add it to `ShowConfig` (in `api_schemas.py`) to complete this.

Otherwise it'll not be available via API.